### PR TITLE
[wasm] refactor debug proxy

### PR DESCRIPTION
**Pull request by lewing**
Remove some unused code and clean up some of the remaining code to use slightly more idiomatic  patterns.

### CODE REVIEW

File: sdks/wasm/ProxyDriver/Program.cs [hunk 1]
```diff
@@ -14,21 +14,42 @@
...
+		public string ChromePath { get; set; }
+		public string AppPath { get; set; }
+		public string PagePath { get; set; }
+		public string NodeApp { get; set; }
+	}
+
+	public class Program
+	{
+		public static void Main(string[] args)
         {
```
Comment (kjpou1) [inline sdks/wasm/ProxyDriver/Program.cs]: nit.  indentation

Comment (lewing) [reply sdks/wasm/ProxyDriver/Program.cs]: 👍

**Comment by lewing** [comment]
@monojenkins build Linux AArch64

**Comment by lewing** [comment]
@monojenkins build failed

**Pull request merged**